tests/js/spec/components/repositoryRow.spec.jsx

```diff
@@ -136,4 +147,49 @@ describe('RepositoryRow', function () {
       expect(cancel).toHaveBeenCalled();
     });
   });
+
+  describe('renders custom_scm repo', function () {
```

Comment:
@MeredithAnya doesn't this test need the feature flag?

Reply:
@NisanthanNanthakumar it wasn't needed when I was only checking the repository provider. But I've updated the tests now that I've added the HOC